src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -1,20 +1,69 @@
-import React from "react";
+import { error } from "console";
+import React, { useState } from "react";
 import "./ModalWindow.css";
+import axios from "axios";
+import { useEffect } from "react";
+import { ReactComponent as Cross } from "../../assets/cross.svg";
 
 interface ModalWindowProps {
   setModalActive: React.Dispatch<React.SetStateAction<boolean>>;
 }
 
+type searchResalt = Array<searchRepoType>;
+
+type searchRepoType = {
```

Comment:
capitalized first letter for types or interfaces

Reply:
Repository

Reply:
DONE